float64[] q_target
float64[] qd_target
float64[] i_target
float64[] m_target
float64[] tau_target
float64[] tool_vector_target
float64[] q_actual
float64[] qd_actual
float64[] i_actual
float64[] tau_actual
float64[] tcp_force
float64[] tool_vector_actual
float64[] tcp_speed
float64[] motor_temperatures
float64[] joint_modes
float64 controller_timer
float64[] qdd_target
float64[] qdd_actual
float64[] tool_acc_values
float64 robot_mode
float64 digital_input_bits
float64 test_value